tools/gcp/comp_test/run_comp_test.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python2.7
```

Comment:
I totally agree there's some actions that need to be performed to 1. prepare backwards compatibility interop tests to be able to run them 2. to actually run them on a selected list of versions.  The problem is that your scripts don't address what I believe is the most interesting part of the problem: the management of the existing versions to test and deciding what tests to run against what. I have trouble believing that just dumping docker images to GCR and then running tests against what's found in GCR (after querying) is a manageable way to tests things - you'll quickly lose track of what are the versions you're testing against (and the version list is going to differ for different kinds of tests e.g. after you find out some specific test in an older version is failing and you don't what that to pollute your test results) - so you will probably need some version controlled script/configuration file to decide what things to run when. Once we have that part, decisions about other parts of the problem (like if we should include specific files inside the docker images or not) will become much more straightforward.